docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+# DALi Widget Application
+
+
+You can create a widget applications with DALi to display small version of application on the homescreen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> A DALi widget application is implemented similarly to a [DALi basic UI application](dali-basic-app.md), since the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class. Consequently, make sure you are familiar with the basic UI application details before tackling a widget application.
+
+- Make sure you [start the event loop](#mainloop).
+
+- Implement your own widget class by inheriting the [Dali::Widget class](#widget-api).
```

Comment:
Implement your own widget class by inheriting the [Dali::Widget class](#widget-api).

In this sentence remove hyperlink from class.